pkg/k8shandler/common.go

```diff
@@ -174,7 +174,6 @@ func newElasticsearchContainer(imageName string, envVars []v1.EnvVar, resourceRe
 				Protocol:      v1.ProtocolTCP,
 			},
 			v1.ContainerPort{
-				Name:          "restapi",
```

Comment:
why did this get removed/moved?

Reply:
This is because we need the service to map to the restapi named port which is the proxy and no longer the ES container. so flow is  service->proxy->localhost->es

Reply:
do we even need the `9200` container port then?

Reply:
leaving based on our discussion